### PR TITLE
AzureMonitor: Update documentation and v8 Whats New

### PR DESCRIPTION
**Pull request by joshhunt**
**What this PR does / why we need it**:

"MVP" documentation and What's New update for 8.0

**Special notes for your reviewer**:

Note, this isn't the full documentation refresh that Azure needs. We'll work on that iteratively, but this is the most important stuff for launch

### CODE REVIEW

File: docs/sources/datasources/azuremonitor.md [hunk 1]
```diff
@@ -10,49 +10,38 @@ weight = 300
...
-- **[Application Insights]({{< relref "#querying-the-application-insights-service" >}})** is an extensible Application Performance Management (APM) service for web developers on multiple platforms and can be used to monitor your live web application - it will automatically detect performance anomalies.
-- **[Application Insights Analytics]({{< relref "#query-the-application-insights-analytics-service" >}})** allows you to query [Application Insights data](https://docs.microsoft.com/en-us/azure/azure-monitor/app/analytics) using the same query language used for Azure Log Analytics.
+- **[Azure Monitor Metrics]({{< relref "#query-the-metrics-service" >}})** (or Metrics) is the platform service that provides a single source for monitoring Azure resources.
+- **[Azure Monitor Logs]({{< relref "#query-the-logs-service" >}})** (or Logs) gives you access to log data collected by Azure Monitor. -**[Azure Monitor Logs]({{< relref "#query-the-azure-resource-graph-service" >}})** (or Logs) gives you access to log data collected by Azure Monitor.
 
 ## Add the data source
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/azuremonitor.md:16]:
```suggestion
## Access Azure metrics
```

File: docs/sources/datasources/azuremonitor.md [hunk 2]
```diff
@@ -10,49 +10,38 @@ weight = 300
...
 
 The data source can access metrics from four different services. You can configure access to the services that you use. It is also possible to use the same credentials for multiple services if that is how you have set it up in Azure AD.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/azuremonitor.md:18]:
```suggestion
The Azure Monitor data source can access metrics from four different services. Configure access to the services that you plan to use. You can use the same credentials for multiple services if that is how you have set it up in Azure AD.
```

**Comment by joshhunt** [comment]
Images in this website PR https://github.com/grafana/website/pull/4656

File: docs/sources/datasources/azuremonitor.md [hunk 3]
```diff
@@ -10,49 +10,38 @@ weight = 300
...
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/azuremonitor.md:19]: Here are some links on how to set up access to the different services:

File: docs/sources/datasources/azuremonitor.md [hunk 4]
```diff
@@ -10,49 +10,38 @@ weight = 300
...
 
 - [Guide to setting up an Azure Active Directory Application for Azure Monitor.](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal)
 - [Guide to setting up an Azure Active Directory Application for Azure Monitor Logs.](https://dev.loganalytics.io/documentation/Authorization/AAD-Setup)
-- [Quickstart Guide for Application Insights.](https://dev.applicationinsights.io/quickstart/)
 
 1. Accessed from the Grafana main menu, newly installed data sources can be added immediately within the Data Sources section. Next, click the "Add data source" button in the upper right. The Azure Monitor data source will be available for selection in the Cloud section in the list of data sources.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/azuremonitor.md:23]: Rewite this following Prometheus data source doc.

File: docs/sources/datasources/azuremonitor.md [hunk 5]
```diff
@@ -10,49 +10,38 @@ weight = 300
...
-1. In the name field, Grafana will automatically fill in a name for the data source - `Azure Monitor` or something like `Azure Monitor - 3`. If you are going to configure multiple data sources, then change the name to something more informative.
+2. In the name field, Grafana will automatically fill in a name for the data source - `Azure Monitor` or something like `Azure Monitor - 3`. If you are going to configure multiple data sources, then change the name to something more informative.
 
-1. If you are using Azure Monitor, you need 4 pieces of information from the Azure portal (see link above for detailed instructions):
+3. If you are using Azure Monitor, you need 4 pieces of information from the Azure portal (see link above for detailed instructions):
 
    - **Tenant Id** (Azure Active Directory -> Properties -> Directory ID)
    - **Client Id** (Azure Active Directory -> App Registrations -> Choose your app -> Application ID)
    - **Client Secret** (Azure Active Directory -> App Registrations -> Choose your app -> Keys)
    - **Default Subscription Id** (Subscriptions -> Choose subscription -> Overview -> Subscription ID)
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/azuremonitor.md:32]: All this information should ideally be in a table.

File: docs/sources/datasources/azuremonitor.md [hunk 6]
```diff
@@ -302,128 +287,15 @@ If you're not currently logged in to the Azure Portal, then the link opens the l
 
 <div class="clearfix"></div>
 
-### Logs alerting
-
-> Only available in Grafana v7.0+.
-
-Grafana alerting is supported for Application Insights. This is not Azure Alerts support. Read more about how alerting in Grafana works in [Alerting rules]({{< relref "../alerting/_index.md" >}}).
-
-## Query Application Insights service
```
Comment (joshhunt) [inline docs/sources/datasources/azuremonitor.md:311]: Application Insights and Insights analytics have been deprecated and cannot be used in new panels

File: docs/sources/datasources/azuremonitor.md [hunk 7]
```diff
@@ -10,49 +10,38 @@ weight = 300
...
-1. If you are using Application Insights, then you need two pieces of information from the Azure Portal (see link above for detailed instructions):
-
-   - Application ID
-   - API Key
-
-1. Paste these two items into the appropriate fields in the Application Insights API Details section:
-   {{< figure src="/static/img/docs/v62/config_2_app_insights_api_details.png" class="docs-image--no-shadow" caption="Application Insights Configuration Details" >}}
-
-1. Test that the configuration details are correct by clicking on the "Save & Test" button:
+6. Test that the configuration details are correct by clicking on the "Save & Test" button:
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/azuremonitor.md:44]:
```suggestion
1. Test that the configuration details are correct by clicking on the "Save & Test" button:
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added some comments, will continue.

File: docs/sources/datasources/azuremonitor.md (hunk 2 above)
Comment (joshhunt) [reply docs/sources/datasources/azuremonitor.md:18]: Ahh, this is actually slightly out of date so I'll change this to 

> The Azure Monitor data source can access metrics from three different services. Configure access to the services that you plan to use. To use different credentials for different Azure services, configure multiple Azure Monitor data sources.

File: docs/sources/datasources/azuremonitor.md (hunk 7 above)
Comment (joshhunt) [reply docs/sources/datasources/azuremonitor.md:44]: I think `6.` is correct - this is the 6th step in the series, no?

File: docs/sources/datasources/azuremonitor.md (hunk 4 above)
Comment (joshhunt) [reply docs/sources/datasources/azuremonitor.md:23]: I'm holding off on the bigger refresh, but rather just making minimum updates to make sure it's accurate for the changes in 8.0.

Next week we'll take a look at refreshing the entire documentation.

File: docs/sources/datasources/azuremonitor.md (hunk 7 above)
Comment (achatterjee-grafana) [reply docs/sources/datasources/azuremonitor.md:44]: All numbered steps should be "1.", they get converted.

File: docs/sources/datasources/azuremonitor.md (hunk 4 above)
Comment (achatterjee-grafana) [reply docs/sources/datasources/azuremonitor.md:23]: Ok.

File: docs/sources/datasources/azuremonitor.md [hunk 8]
```diff
@@ -63,19 +52,15 @@ az ad sp create-for-rbac -n "http://localhost:3000"
...
+- Metrics
+- Logs
+- Azure Resource Graph
 
 The query editor changes depending on which one you pick. Metrics is the default.
 
-Starting in Grafana 7.1, Insights Analytics replaced the former edit mode from within Application Insights.
-
-Starting in Grafana 7.4, the Azure Monitor query type was renamed to Metrics and Azure Logs Analytics was renamed to Logs.
+Starting in Grafana 7.4, the Azure Monitor query type was renamed to Metrics and Azure Logs Analytics was renamed to Logs. Starting in Grafana 8.0 Application Insights and Insights Analytics is unavailable for new panels, in favor of querying through Metrics and Logs.
```
Comment (oddlittlebird) [inline docs/sources/datasources/azuremonitor.md:63]: @achatterjee-grafana, what do you think of putting this paragraph in a note?

File: docs/sources/whatsnew/whats-new-in-v8-0.md [hunk 9]
```diff
@@ -163,6 +162,57 @@ These improvements affect any subsequent data updates or interactions, including
 
 The following data source updates are included with this Grafana release.
 
+#### Azure Monitor data source
+
+Grafana 8.0 includes some major changes and new functionality to make using and configuring the Azure Monitor data source easier and simpler, and to pave the way for future improvements.
```
Comment (oddlittlebird) [inline docs/sources/whatsnew/whats-new-in-v8-0.md:167]: Add a link in this paragraph about how the Azure Monitor data source topic was updated as a result of these changes. 

Also, keep in mind that the point of the What's new doc is to tell the user WHAT has changed and to provide a link to the docs with the details of HOW it has changed. Try to keep this content sparse, especially if it is likely to get stale in future releases.

Comment (achatterjee-grafana) [reply docs/sources/whatsnew/whats-new-in-v8-0.md:167]: Had a chat with @joshhunt, I will create a separate PR for the What's New content. He will refine this PR next week.

File: docs/sources/datasources/azuremonitor.md [hunk 10]
```diff
@@ -10,49 +10,38 @@ weight = 300
...
-- **[Application Insights]({{< relref "#querying-the-application-insights-service" >}})** is an extensible Application Performance Management (APM) service for web developers on multiple platforms and can be used to monitor your live web application - it will automatically detect performance anomalies.
-- **[Application Insights Analytics]({{< relref "#query-the-application-insights-analytics-service" >}})** allows you to query [Application Insights data](https://docs.microsoft.com/en-us/azure/azure-monitor/app/analytics) using the same query language used for Azure Log Analytics.
+- **[Azure Monitor Metrics]({{< relref "#query-the-metrics-service" >}})** (or Metrics) is the platform service that provides a single source for monitoring Azure resources.
+- **[Azure Monitor Logs]({{< relref "#query-the-logs-service" >}})** (or Logs) gives you access to log data collected by Azure Monitor. -**[Azure Monitor Logs]({{< relref "#query-the-azure-resource-graph-service" >}})** (or Logs) gives you access to log data collected by Azure Monitor.
```
Comment (sarahzinger) [inline docs/sources/datasources/azuremonitor.md:14]: It looks like you have this Azure Monitor Logs part twice with 2 different links is that intentional?

**Review by sarahzinger — COMMENTED** [review]
cool stuff! I'm not able to build the docs locally for some reason, but I'm pretty sure thats something wrong with my own local setup and unrelated to this pr but the result is I can't view this outside of github at the moment. 

Left a few comments, just catching typos

File: docs/sources/datasources/azuremonitor.md (hunk 10 above)
Comment (joshhunt) [reply docs/sources/datasources/azuremonitor.md]: Good catch!

File: docs/sources/datasources/azuremonitor.md [hunk 11]
```diff
@@ -10,49 +10,38 @@ weight = 300
...
    - **Default Subscription Id** (Subscriptions -> Choose subscription -> Overview -> Subscription ID)
 
-1. Paste these four items into the fields in the Azure Monitor API Details section:
+4. Paste these four items into the fields in the Azure Monitor API Details section:
    {{< figure src="/static/img/docs/v62/config_1_azure_monitor_details.png" class="docs-image--no-shadow" caption="Azure Monitor Configuration Details" >}}
 
    - The Subscription Id can be changed per query. Save the data source and refresh the page to see the list of subscriptions available for the specified Client Id.
 
-1. If you are also using the Azure Monitor Logs service, then you need to specify these two configuration values (or you can reuse the Client Id and Secret from the previous step).
+5. If you are also using the Azure Monitor Logs service, then you need to specify these two configuration values (or you can reuse the Client Id and Secret from the previous step).
```
[the remark this answers is not in the document]
Comment (joshhunt) [reply docs/sources/datasources/azuremonitor.md]: Good catch. Removed this 😎

**Comment by joshhunt** [comment]
@achatterjee-grafana I've updated this PR with some slight changes to the What's New (mentioned Managed Identity, which is pretty big/important) and cleaned up azure docs a bit more.

I'm going to work on full on refresh of Azure documentation separately, next.

**Comment by achatterjee-grafana** [comment]
@joshhunt, the What's New section was already updated. We need to remove it from this PR. Thanks!

**Comment by joshhunt** [comment]
@achatterjee-grafana  I know, I just added Managed Identity to the What's New with some minor changes to what you added on Friday, and updated the Azure Monitor documentation.

File: docs/sources/datasources/azuremonitor.md [hunk 12]
```diff
@@ -63,19 +48,15 @@ az ad sp create-for-rbac -n "http://localhost:3000"
...
+- Metrics
+- Logs
+- Azure Resource Graph
 
 The query editor changes depending on which one you pick. Metrics is the default.
 
-Starting in Grafana 7.1, Insights Analytics replaced the former edit mode from within Application Insights.
-
-Starting in Grafana 7.4, the Azure Monitor query type was renamed to Metrics and Azure Logs Analytics was renamed to Logs.
+Starting in Grafana 7.4, the Azure Monitor query type was renamed to Metrics and Azure Logs Analytics was renamed to Logs. Starting in Grafana 8.0 Application Insights and Insights Analytics is unavailable for new panels, in favor of querying through Metrics and Logs.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/azuremonitor.md:59]:
```suggestion
In Grafana 7.4, the Azure Monitor query type was renamed to Metrics, and Azure Logs Analytics was renamed to Logs. In Grafana 8.0 Application Insights and Insights Analytics is unavailable for new panels, in favor of querying through Metrics and Logs.
```

File: docs/sources/datasources/azuremonitor.md [hunk 13]
```diff
@@ -302,128 +283,15 @@ If you're not currently logged in to the Azure Portal, then the link opens the l
...
 
-{{< figure src="/static/img/docs/azure-monitor/azure-resource-graph-8-0.png" class="docs-image--no-shadow" caption="Azure Resource Graph Editor" >}}
+{{< figure src="/static/img/docs/azure-monitor/azure-resource-graph.png" class="docs-image--no-shadow" caption="Azure Resource Graph editor" max-width="650px" >}}
 
 ### Table queries
 
-Queries are written in the [Kusto Query Language](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/). An Azure Resource Graph query is formatted as table data.
+Queries are written in the [Kusto Query Language](https://docs.microsoft.com/en-us/azure/governance/resource-graph/concepts/query-language). Please note that not all Kusto language features are available in ARG. An Azure Resource Graph query is formatted as table data.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/azuremonitor.md:294]:
```suggestion
Queries are written in the [Kusto Query Language](https://docs.microsoft.com/en-us/azure/governance/resource-graph/concepts/query-language). Not all Kusto language features are available in ARG. An Azure Resource Graph query is formatted as table data.
```

File: docs/sources/datasources/azuremonitor.md [hunk 14]
```diff
@@ -476,3 +344,89 @@ securityresources
     portalLink=properties.links.azurePortal
 | summarize numberOfResources=count(resourceId) by tostring(recommendationName), tostring(recommendationState)
 ```
+
+## Configure the data source with provisioning
+
+You can configure data sources using config files with Grafana’s provisioning system. You can read more about how it works and all the settings you can set for data sources on the [provisioning docs page]({{< relref "../administration/provisioning/#datasources" >}})
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/azuremonitor.md:350]:
```suggestion
You can configure data sources using config files with Grafana’s provisioning system. For more information on how it works and all the settings you can set for data sources on the [provisioning docs page]({{< relref "../administration/provisioning/#datasources" >}})
```

File: docs/sources/datasources/azuremonitor.md [hunk 15]
```diff
@@ -476,3 +344,89 @@ securityresources
...
+### Supported cloud names
+
+| Azure Cloud                                      | Value                      |
+| ------------------------------------------------ | -------------------------- |
+| Microsoft Azure public cloud                     | `azuremonitor` (_default_) |
+| Microsoft Chinese national cloud                 | `chinaazuremonitor`        |
+| US Government cloud                              | `govazuremonitor`          |
+| Microsoft German national cloud ("Black Forest") | `germanyazuremonitor`      |
+
+## Deprecating Application Insights and Insights Analytics
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/azuremonitor.md:418]:
```suggestion
## Deprecated Application Insights and Insights Analytics
```

File: docs/sources/datasources/azuremonitor.md [hunk 16]
```diff
@@ -476,3 +344,89 @@ securityresources
...
+
+For Application Insights, new queries can be made with the Metrics query type by selecting the "Application Insights" resource type.
+
+{{< figure src="/static/img/docs/azure-monitor/app-insights-metrics.png" max-width="650px" class="docs-image--no-shadow" caption="Azure Monitor Application Insights example" >}}
+
+For Insights Analaytics, new queries can be written with Kusto in the Logs query type by selecting your Application Insights resource.
+
+{{< figure src="/static/img/docs/azure-monitor/app-insights-logs.png" max-width="650px" class="docs-image--no-shadow" caption="Azure Logs Application Insights example" >}}
+
+{{< figure src="/static/img/docs/azure-monitor/app-insights-resource-picker.png" max-width="650px" class="docs-image--no-shadow" caption="Azure Logs Application Insights resource picker" >}}
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/azuremonitor.md:430]: Here you have 2 images back to back, is this by design? Needs a sentence in between explaining the second image.

File: docs/sources/datasources/azuremonitor.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/datasources/azuremonitor.md:17]: I do not see this change, I had requested earlier.

**Review by achatterjee-grafana — COMMENTED** [review]
Added some review comments. In general looks much cleaner, thank you for the re-org. We can continue with improvements.

File: docs/sources/datasources/azuremonitor.md (hunk 1 above)
Comment (joshhunt) [reply docs/sources/datasources/azuremonitor.md:17]: I'm not too sure this change makes sense? This section is about adding and configuring Azure Monitor

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**